src/main/java/view_post/use_case/ViewPostResponseModel.java

```diff
@@ -19,12 +25,25 @@ public class ViewPostResponseModel {
      * @param deadline      The string representation of the deadline of this post.
      */
     public ViewPostResponseModel(String posterEmail, String postBody, String[] postTags, String[] postReplies,
-                                 String deadline) {
+                                 String deadline, String creationDate, String collaborators, int postID, String title) {
         this.posterEmail = posterEmail;
         this.postBody = postBody;
-        this.postTags = postTags;
-        this.postReplies = postReplies;
+        String tags = "";
+        for (String tag : postTags){
+            tags.concat(", ");
```

Comment:
this needs to be tags = tags.concat(", ") otherwise it does not assign the concatenation to anything!